ndb/src/google/cloud/ndb/client.py

```diff
@@ -70,6 +70,9 @@ class Client(ClientWithProject):
     SCOPE = ("https://www.googleapis.com/auth/datastore",)
     """The scopes required for authenticating as a Cloud Datastore consumer."""
 
+    secure = True
```

Comment:
```suggestion
    SECURE = True
```

What is the genesis / need of this class attribute?

Reply:
https://github.com/googleapis/google-cloud-python/blob/572b3df37c3e66d5a59e279c0e71838254212625/datastore/google/cloud/datastore/_gapic.py#L39

Datastore selects a secure channel or an insecure channel for gRPC based on the `base_url`--if it starts with "https", it creates a secure channel. Since we're just using `host` and not `base_url`, I added a boolean flag. It's meant to be overridden on a per-instance basis, like `base_url` in Datastore. I'm not sure what the use case is for Datastore for overriding the default `base_url` (testing? read-only access? non-Google Datastore servers?), but I figured I needed to preserve it, in spirit, by allowing the host and secure flag to be overridden.

Reply:
Why does it need to be a class attribute then?

Reply:
FWIW, using class attributes as defaults seems particularly sane in a library which already abuses the snot out of `__new__`. :)